docs/resources/dli_queue.md

```diff
@@ -13,32 +15,63 @@ This is an alternative to `huaweicloud_dli_queue`
 
 ```hcl
 resource "huaweicloud_dli_queue" "queue" {
-  name     = "terraform_dli_queue_test"
-  cu_count = 4
+  name          = "terraform_dli_queue_test"
+  cu_count      = 16
+  resource_mode = 0
```

Comment:
there is unnecessary to specifie resource_mode, as it is optional and defaults to 0.